src/metabase/channel/render/card.clj

```diff
@@ -145,6 +149,26 @@
           (log/error e "Pulse card render error")
           (body/render :render-error nil nil nil nil nil))))))
 
+(def branding-logo-base64-src
+  "Returns a Base64 data URI of a PNG logo from assets."
+  (->
+   (io/resource "frontend_client/app/assets/img/exports-branding-logo.png")
+   io/input-stream
+   IOUtils/toByteArray
+   image-bundle/render-img-data-uri))
+
+(def branding-content
+  "Generates a branding content that we send below charts in dashboard subscriptions and Slack notifications."
+  [:table {:style (style/style {:height :32px :width "100%" :font-size :12px})}
+   [:tr {}
+    [:td {:style (style/style {:width "100%" :vertical-align :middle :padding-right :8px :text-align :right})} "Made with"]
+    [:td {:style (style/style {:vertical-align :middle})}
+     [:img {:src branding-logo-base64-src :max-width :81px}]]]])
+
+(def include-branding?
+  "Branding in exports is included only for instances that do not have a whitelabel feature flag."
```

Comment:
`(metabase.premium-features.core/enable-whitelabeling?)` should be using this. I still am wondering if we intend to show our logo for paid customers who lack whitelabeling.

Reply:
https://github.com/metabase/metabase/pull/57434/commits/4fb43f0f0c0e3546ad894cda849397ca91e2520e